lib/ReactViews/Map/Panels/SharePanel/BuildShareLink.js

```diff
@@ -95,6 +95,8 @@ function addStratum(terria, stratumId, initSource) {
   });
 }
 
+function addBaseMaps(terria, initSource) {}
+
```

Comment:
If you end up moving `TerriaViewer.basemaps` into `BaseMapsModel` - you will need to use something like `addModelStratum()` to add `basemaps` model to the `initSource`